AyraaCollections.Logic/BusinessLogics/AuthenticationBL.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Linq;
+using AyraaCollections.Common;
+using AyraaCollections.Data;
+
+public class AuthenticationBL
+{
+    public bool Register(UserDTO dtoUser)
+    {
+
+        User user = new User(dtoUser);
```

Comment:
Check user is there or not already - using email?

Reply:
changed.